pubsub/subscriptions/main.go

```diff
@@ -167,6 +170,37 @@ func pullMsgsSettings(client *pubsub.Client, subName string) error {
 	return nil
 }
 
+func pullMsgsConcurrencyControl(client *pubsub.Client, subName string, numGoroutines int) error {
+	ctx := context.Background()
+	// [START pubsub_subscriber_concurrency_control]
+	sub := client.Subscription(subName)
+	sub.ReceiveSettings.NumGoroutines = numGoroutines
+	err := sub.Receive(ctx, func(ctx context.Context, msg *pubsub.Message) {
+		fmt.Printf("Got message %q\n", string(msg.Data))
+		msg.Ack()
+	})
+	if err != nil {
+		return err
```

Comment:
```suggestion
		return fmt.Errorf("Receive: %v", err)
```

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#return-errors